site/templates/footer.html

```diff
@@ -4,6 +4,6 @@
 
 <link href='https://fonts.googleapis.com/css?family=Quicksand:300,400' rel='stylesheet' type='text/css'>
 <link href='https://fonts.googleapis.com/css?family=Oswald' rel='stylesheet' type='text/css'>
-<script src="/js/main.js"></script>
+<script src="js/main.js"></script>
```

Comment:
Here's one of those example cases, I think we may want `{{baseUrl}}/js/main.js` instead right?

Reply:
Yeah, I don't think my change here is a good idea. What was happening is I was trying to make this work if it's hosted at all these shapes of base with the same unchanged pages. But I think this is unrealistic.

- some.domain.tld
- some.domain.tld/subdir  <-- this one is tricky, because the `/images/foo.png` types of links will not get that subdir
- localhost:8000
- 192.168.1.10:8000

Some of these are entirely to support viewing the site while hacking on it or previewing pages and this is not a good reason to make a change like this.
I removed the `0550244 Now using relative urls and a base element` commit from this PR entirely.

Sorry about that and thanks for discussing!

Reply:
What did you think of using the baseUrl in these links as I suggested? E.g. `{{baseUrl}}/js/main.js`
I would like to have some fix so that people can host on arbitrary paths.

Reply:
I'm no expert but it seems like mucking around with some of this in URLs (also adding a `<head><base href="http://mydomain/" />` breaks working locally without deploying to the "production" server where the domain is. As in, using `npm serve` or similar and hitting `localhost:8000`. I think leaving it like it is works better for my purposes.